src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/LogicalExpression.java

```diff
@@ -123,6 +104,7 @@ public void addLogicalExpression(LogicalExpression logicalExpression) {
     logicalExpressions.add(logicalExpression);
   }
 
+  // TODO: this is the same function as below, this is just code duplication
```

Comment:
Yuqi Resolve: delete the duplication, all use the addComparisonExpressions method